app/src/components/common/outcomes/index.tsx

```diff
@@ -173,9 +209,46 @@ const Outcomes = (props: Props) => {
     )
   }
 
+  const addNewOutcome = () => {
```

Comment:
There's a lot of duplication in these methods. A helper `uniform(outcomes)` should help.